### PR TITLE
Changelog for 2.3.6 

### PR DESCRIPTION
**Pull request by daschuer**
I have update the changelog for the upcoming 2.3.6 release. The last release before we release the 2.4.0. 

Please have also a look into with some waiting PRs: 
https://github.com/mixxxdj/mixxx/milestone/40

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,19 +2,30 @@
...
-  [#11595](https://github.com/mixxxdj/mixxx/pull/11595)
-* No longer stop a track with an active loop at the very end.
-  [#11558](https://github.com/mixxxdj/mixxx/pull/11558)
-  [#11557](https://github.com/mixxxdj/mixxx/issues/11557)
-* Shade: Fixed VU-Meter and other minor issues
-  [#11598](https://github.com/mixxxdj/mixxx/pull/11598)
 * Fixed possible crash when closing Mixxx while browsing the file system
   [#11593](https://github.com/mixxxdj/mixxx/pull/11593)
   [#11589](https://github.com/mixxxdj/mixxx/issues/11589)
+* No longer stop a track with an active loop at the very end.
```
Comment (ronso0) [inline CHANGELOG.md:8]:
```suggestion
* No longer stop a track with an active loop at the very end
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -2,19 +2,30 @@
...
+  [#11661](https://github.com/mixxxdj/mixxx/pull/11661)
+* Console log spam reduced
+  [#11690](https://github.com/mixxxdj/mixxx/issues/11690)
+  [#11691](https://github.com/mixxxdj/mixxx/issues/11691)
+* Numark DJ2GO2 Touch: Add missing loop_out mapping for the right deck
+  [#11595](https://github.com/mixxxdj/mixxx/pull/11595)
+  [#11659](https://github.com/mixxxdj/mixxx/pull/11659)
+* Shade: Fixed VU-Meter and other minor issues
+  [#11598](https://github.com/mixxxdj/mixxx/pull/11598)
+* LateNight PaleMoon: Tweak style
```
Comment (ronso0) [inline CHANGELOG.md:28]:
```suggestion
```
I didn't find a related PR, maybe I pushed some minor fixes directly?
Not sure, so let's remove it.

**Review by ronso0 — CHANGES_REQUESTED** [review]
Thanks!
just some small fixes, didn't check the PR links, yet

File: CHANGELOG.md (hunk 2 above)
Comment (daschuer) [reply CHANGELOG.md:28]: This was a direct push without a PR. I will remove it.

**Comment by daschuer** [comment]
Done

**Comment by daschuer** [comment]
This is now again up to date. There are 10 entries in the 2.3 milestone. It would be nice if we get some of the merged before a release.

File: CHANGELOG.md [hunk 3]
```diff
@@ -2,19 +2,34 @@
...
 * Fixed possible crash when closing Mixxx while browsing the file system
   [#11593](https://github.com/mixxxdj/mixxx/pull/11593)
   [#11589](https://github.com/mixxxdj/mixxx/issues/11589)
+* No longer stop a track with an active loop at the very end
+  [#11558](https://github.com/mixxxdj/mixxx/pull/11558)
+  [#11557](https://github.com/mixxxdj/mixxx/issues/11557)
 * Fixed resyncing when moving an active loop
   [#11152](https://github.com/mixxxdj/mixxx/pull/11152)
   [#11381](https://github.com/mixxxdj/mixxx/issues/11381)
+* Fixed gapless repeat of full tracks
```
Comment (ronso0) [inline CHANGELOG.md:14]:
```suggestion
* Allow true gapless playback when repeating full tracks
```

File: CHANGELOG.md [hunk 4]
```diff
@@ -2,19 +2,34 @@
...
+  [#11532](https://github.com/mixxxdj/mixxx/pull/11532)
+  [#9842](https://github.com/mixxxdj/mixxx/issues/9842)
+  [#11704](https://github.com/mixxxdj/mixxx/pull/11704)
+* Rhythmbox: Fixed bulk track imports from playlists
+  [#11661](https://github.com/mixxxdj/mixxx/pull/11661)
+* Console log spam reduced
+  [#11690](https://github.com/mixxxdj/mixxx/issues/11690)
```
Comment (ronso0) [inline CHANGELOG.md:21]: it's redirecting but why not
```suggestion
  [#11690](https://github.com/mixxxdj/mixxx/pull/11690)
```

File: CHANGELOG.md [hunk 5]
```diff
@@ -2,19 +2,34 @@
...
+* Numark DJ2GO2 Touch: Add missing loop_out mapping for the right deck
+  [#11595](https://github.com/mixxxdj/mixxx/pull/11595)
+  [#11659](https://github.com/mixxxdj/mixxx/pull/11659)
+* Shade: Fixed VU-Meter and other minor issues
+  [#11598](https://github.com/mixxxdj/mixxx/pull/11598)
+* Fixed a rare crash when disabling quantize form a controller
+  [#11744](https://github.com/mixxxdj/mixxx/pull/11744)
+  [#11709](https://github.com/mixxxdj/mixxx/issues/11709)
+* Controller Preferences: Avoided a scrollbars in I/O tabs if Info tab exceeds page height
+  [#11756](https://github.com/mixxxdj/mixxx/issues/11756)
```
Comment (ronso0) [inline CHANGELOG.md:32]:
```suggestion
  [#11756](https://github.com/mixxxdj/mixxx/pull/11756)
```

**Review by ronso0 — CHANGES_REQUESTED** [review]
A bit of polishing, else LGTM

**Comment by daschuer** [comment]
Done

File: CHANGELOG.md [hunk 6]
```diff
@@ -2,19 +2,34 @@
...
+  [#11691](https://github.com/mixxxdj/mixxx/issues/11691)
+* Numark DJ2GO2 Touch: Add missing loop_out mapping for the right deck
+  [#11595](https://github.com/mixxxdj/mixxx/pull/11595)
+  [#11659](https://github.com/mixxxdj/mixxx/pull/11659)
+* Shade: Fixed VU-Meter and other minor issues
+  [#11598](https://github.com/mixxxdj/mixxx/pull/11598)
+* Fixed a rare crash when disabling quantize form a controller
+  [#11744](https://github.com/mixxxdj/mixxx/pull/11744)
+  [#11709](https://github.com/mixxxdj/mixxx/issues/11709)
+* Controller Preferences: Avoided a scrollbars in I/O tabs if Info tab exceeds page height
```
Comment (ronso0) [inline CHANGELOG.md:31]: Just one more typo, then this is good to go 👍
```suggestion
* Controller Preferences: Avoid scrollbars in I/O tabs if Info tab exceeds page height
```

**Comment by daschuer** [comment]
Done again. Thank you for review.

**Comment by ronso0** [comment]
Thank you, LGTM

**Pull request merged**